app/uk/gov/hmrc/cataloguefrontend/connector/PrCommenterConnector.scala

```diff
@@ -38,7 +39,7 @@ object PrCommenterConnector {
     created  : Instant
   ) {
     // Repos with joint ownership of 8+ teams, defines the repo as shared
```

Comment:
probably don't need the comment (we shouldn't maintain the number in both comment and constant)